python/ml4ir/base/stats/t_test.py

```diff
@@ -458,6 +458,16 @@ def compute_required_sample_size(mean1, mean2, var1, var2, statistical_power, pv
         if denominator == 0:
             return np.inf
         d = np.abs(float(mean1) - float(mean2)) / denominator
+        '''
```

Comment:
Minor Questions: 
- Should we be using `"` instead of `'` as in the docstring? 
- There seems to be an extra indent in the details compared to the `'''`
- Please start with capital (`For`)
- Typos: `temporeary`

Main questions: 
- How is 2.24 calculated? 
- What's the plan to actually fix the issue and remove this temporary fix? Is there one? 
- power==0.9 and pvalue==0.1 are very specifix. Should we have something more generic e.g., `power>=0.9` and `pvalue<=0.1`?    

Thanks for catching this and trying to fix it here. Is there already an open issue on `power_ttest`?